### PR TITLE
Update French Language

### PR DESCRIPTION
**Pull request by bubuleur**

### CODE REVIEW

**Comment by cgutman** [comment]
Can you also translate `title_checkbox_vibrate_fallback` and `summary_checkbox_vibrate_fallback`?

**Comment by bubuleur** [comment]
sorry I did not see him thanks to you

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -147,6 +150,8 @@
     <string name="category_on_screen_controls_settings">Paramètres des contrôles à l\'écran</string>
     <string name="title_checkbox_show_onscreen_controls">Afficher les commandes à l\'écran</string>
     <string name="summary_checkbox_show_onscreen_controls">Afficher la superposition du contrôleur virtuel sur l\'écran tactile</string>
+    <string name="title_checkbox_vibrate_osc">Activer les vibrations</string>
+    <string name="summary_checkbox_vibrate_osc">Active l'émulation sur votre appareil des vibrations des commandes à l'écran</string>
```
Comment (cgutman) [inline app/src/main/res/values-fr/strings.xml]: All `'` characters need to have `\` before them, like `\'`

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -133,6 +134,8 @@
     <string name="category_input_settings">Paramètres d\'entrée</string>
     <string name="title_checkbox_multi_controller">Prise en charge de plusieurs contrôleurs</string>
     <string name="summary_checkbox_multi_controller">Lorsqu\'elle n\'est pas cochée, tous les contrôleurs sont regroupés</string>
+    <string name="title_checkbox_vibrate_fallback">Emuler les vibrations sur votre support</string>
+    <string name="summary_checkbox_vibrate_fallback">Tester si votre appareil émule les vibrations</string>
```
Comment (cgutman) [inline app/src/main/res/values-fr/strings.xml]: I'm not a French speaker so I'm just using Google Translate to read it, but this translation doesn't seem to convey the meaning of the original English text.

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (bubuleur) [reply app/src/main/res/values-fr/strings.xml]: Sorry I had not seen you have the eye ;-)

File: app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (bubuleur) [reply app/src/main/res/values-fr/strings.xml]: Yes it's true I do not really understand the sentence in English at the base
Rumble must it be translated?

Comment (bubuleur) [reply app/src/main/res/values-fr/strings.xml]: Hello, it's good I just changed the problem of translations I just saw that you have released an update at the moment hoping that my translation will be in your next update good day

**Comment by bubuleur** [comment]
2

**Comment by cgutman** [comment]
Thanks

**Pull request merged**